src/index.html

```diff
@@ -7,10 +7,35 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <title>Search bar airbnb</title>
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    <form 
+    action="/" 
+    method="post"
+    data-qa="big">
```

Comment:
```suggestion
    <form 
      action="/" 
      method="post"
      data-qa="big"
    >
```
Зверни увагу на форматування і відступи
Почитати можна тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation